mantidproject/mantidimaging
# COR/Tilt UI redesign

Adds the ability to manually change the COR/Tilt finding results or perform finding by manual inspection of individual reconstructed slices.

To test:
- Load all images from `babylon5/DanNixon/sample_intensity_cutoff_crop/`
- Open *Reconstruct* > *Find COR and Tilt*
- (no cropping is needed as the dataset is already cropped to a correct region)
- Set 45 projections and 10 slices
- Click *Calculate*
- (*Results* tab should be automatically shown)
- Click *Clear All*
- Select somewhere near the top of the projection preview and click *Add*, enter a COR of around 610 (try to remove the halo around the small cylinders)
- Do the last step two or three more times at even intervals down the image
- Click *Fit*
- Click anywhere on the image and you should get a reasonable reconstruction (at least as reasonable as it was when you selected the CORs)
- Right click on the image and view its metadata
- Ensure the COR/Tilt data reflects the last entry in the *Results* tab
